doc/whats_new.rst

```diff
@@ -10,6 +10,10 @@ Fixes
 Enhancements
 ------------
 
+- :func:`nilearn.masking.compute_brain_mask` accepts now as input not only the
+averaged, whole-brain MNI152 T1 template (“avg152T1_brain.nii.gz”) but also
```

Comment:
@alpinho the CircleCI error you're getting is due to the formatting of these lines. You should align them like it is done in the rest of the file.

Reply:
Ah, thanks! I just fixed it.